Survey/Sources/Presentation/Modules/SurveyQuestion/SurveyQuestionView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  SurveyQuestionView.swift
+//  Survey
+//
+//  Created by Khanh on 23/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct SurveyQuestionView: View {
+
+    // TODO: Remove dummy survey and list
+    let survey = APISurvey.dummyList[0]
+    let list = Array(APISurvey.dummyList.enumerated())
+
+    var body: some View {
+        LoadingView(
+            isShowing: .constant(false),
+            text: .constant(""),
+            content: {
+                ZStack {
+                    TabView {
+                        ForEach(list, id: \.element.id) { _ in
+                            SurveyQuestionBodyView()
+                        }
+                    }
+                    .padding(.top, 60.0)
+                    .padding(.bottom, 60.0)
+                    .tabViewStyle(
+                        PageTabViewStyle(
+                            indexDisplayMode: .never
+                        )
+                    )
+                }
+                .overlay {
+                    closeButtonSetup()
+                    nextQuestionButtonSetup()
+                }
+            }
+        )
+        .padding(.bottom, 54.0)
+        .padding(.top, 54.0)
+        .background(
+            // TODO: Remove dummy cover image url
+            Image(survey.coverImageURL)
```

Comment:
Even known this is a dummy data, I think you can reuse `list.first?.coverImageURL` so that you don't have to include a new line of `let`.